src/Core/src/ImageSources/UriImageSourceService/UriImageSourceService.iOS.cs

```diff
@@ -1,16 +1,18 @@
 ﻿#nullable enable
 using System;
+using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
 using Foundation;
 using Microsoft.Extensions.Logging;
-using ObjCRuntime;
 using UIKit;
 
 namespace Microsoft.Maui
 {
 	public partial class UriImageSourceService
 	{
+		internal string CacheDirectory = Essentials.FileSystem.CacheDirectory + "/com.microsoft.maui.sample/MauiUriImages/";
```

Comment:
Not sure if CachDirectory has a trailing /, but we could be extra safe and use Path.Combine() here to build up the path and let the sdk figure out the path separators.

Reply:
done!